sklearn/ensemble/stacking_classifier.py

```diff
@@ -0,0 +1,295 @@
+import numpy as np
+from ..base import BaseEstimator, ClassifierMixin
+from ..base import clone, is_classifier
+from ..model_selection._validation import cross_val_predict
+from ..preprocessing import LabelEncoder
+from ..externals.joblib import Parallel, delayed
+from ..utils.metaestimators import if_delegate_has_method
+from ..utils.validation import has_fit_parameter, check_is_fitted
+from ..externals import six
+
+
+def _parallel_predict(estimator, method, X, y, fit_params):
+    """Private function used to fit an estimator within a job."""
+    estimator.fit(X, y, **fit_params)
+    return estimator, getattr(estimator, method)(X)
+
+
+class StackingClassifier(BaseEstimator, ClassifierMixin):
+    """ Stacking classifier for combining unfitted estimators
+
+    The cross-validated predictions of unfitted estimators will be used as
+    inputs to a meta-estimator for making a final prediction
+
+    Parameters
+    ----------
+    estimators : list of (string, estimator) tuples
+        Invoking the ``fit`` method on the ``StackingClassifier`` will fit
+        clones of those original estimators that will be stored in the class
+        attribute `self.estimators_`.
+
+    meta_estimator : estimator
+        The meta-estimator to combine the predictions of each individual
+        estimator
+
+    method : string, optional, default='predict_proba'
```

Comment:
I think "default" should be "auto" which is predict_proba if it exists and decision_function otherwise.